Docs update (#384)
tendermint/starport: docs/i Appendix/2 Type Scaffolding.md

# Type Scaffolding

> You can scaffold types within Starport by entering in the command:

Reviewer: ```suggestion
You can scaffold types within Starport by running a command:
```

```

## Stargate

This will create the following files:

```
.
├── proto
│   └── module
│       └── v1beta
│           └── {{typeName}}.proto
└── x
    └── module
        ├── client
        │   ├── cli
        │   │   ├── query{{TypeName}}.go
        │   │   └── tx{{TypeName}}.go
        │   └── rest
        │       ├── query{{TypeName}}.go
        │       └── tx{{TypeName}}.go
        ├── keeper
        │   ├── grpc_query_{{typeName}}.go
        │   ├── querier_{{typeName}}.go
        │   └── {{typeName}}.go
        ├── types
        │   ├── MsgCreate{{TypeName}}.go
        │   └── {{typeName}}.pb.go
        └── handlerMsgCreate{{TypeName}}.go
```

As well as update the following files:

```
.
├── proto
│   └── module
│       └── v1beta
│           └── querier.proto
└── x
    └── module
        ├── client
        │   ├── cli
        │   │   ├── query.go
        │   │   └── tx.go
        │   └── rest
        │       └── rest.go
        ├── handler.go
        ├── keeper
        │   └── querier.go
        └── types
            ├── codec.go
            ├── keys.go
            ├── querier.go
            └── querier.pb.go
```

# Launchpad

> Using `starport type` on a Launchpad application will create the follwing files:

Reviewer: ```suggestion
Using `starport type` on a Launchpad application will create the following files:
```
